process/nodeStatusProcessor.go

```diff
@@ -126,6 +132,54 @@ func (nsp *NodeStatusProcessor) GetAllIssuedESDTs() (*data.GenericAPIResponse, e
 	return nil, ErrSendingRequest
 }
 
+// GetDelegatedInfo returns the delegated info from nodes
+func (nsp *NodeStatusProcessor) GetDelegatedInfo() (*data.GenericAPIResponse, error) {
+	observers, err := nsp.proc.GetObservers(core.MetachainShardId)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, observer := range observers {
+		var delegatedInfoResponse *data.GenericAPIResponse
+
+		_, err := nsp.proc.CallGetRestEndPoint(observer.Address, DelegatedInfoPath, &delegatedInfoResponse)
```

Comment:
maybe you can add a cache for these new endpoints.

Reply:
I don't think it's not needed since they are secured